installing/validating-an-installation.adoc

```diff
@@ -0,0 +1,61 @@
+[id="validating-an-installation"]
+= Validating an installation
+include::modules/common-attributes.adoc[]
+:context: validating-an-installation
+
+toc::[]
+
+You can check the status of an {product-title} cluster after an installation by following the procedures in this document.
+
+//Reviewing the installation log
+include::modules/reviewing-the-installation-log.adoc[leveloffset=+1]
+
+//Getting cluster version and update details
+include::modules/getting-cluster-version-and-update-details.adoc[leveloffset=+1]
+
+.Additional resources
```

Comment:
For assemblies, I believe we're doing `==` headers along with an `[id=""]` instead of `.`.

Reply:
But this is a cheat because of the lack of `xref` support? Ignore the previous comment, then.

Reply:
Yes this is an 'Additional resources' section specific to the module that precedes this line.